ern-container-gen-android/src/hull/lib/src/main/java/com/walmartlabs/ern/container/ElectrodeReactContainer.java

```diff
@@ -27,6 +27,7 @@
 import com.facebook.react.ReactPackage;
 import com.facebook.react.bridge.ReactContext;
 import com.facebook.react.bridge.SafeActivityStarter;
+import com.facebook.soloader.SoLoader;
```

Comment:
Would it makes sense to also wrap this import in a conditional? Otherwise it would result in an unused import if `RN_VERSION_GTE_60_1` is false.

Reply:
Yes it would definitely make sense :+1: Adding it